src/devices/Pn5180/Pn5180.cs

```diff
@@ -464,6 +464,31 @@ public override int Transceive(byte targetNumber, ReadOnlySpan<byte> dataToSend,
             }
         }
 
+        /// <inheritdoc/>
+        public override bool ReselectTarget(byte targetNumber)
+        {
+            if (targetNumber == 0)
+            {
+                // TODO: this should be implemented this for Type A card for this reader
+                // This will need to send WUPA (0x52 coded on 7 bits), Anti-collision and select loops like for initial detection
+                // We don't throw an exception, this is just telling that the selection failed
+                return false;
+            }
+            else
+            {
+                var card = _activeSelected.Where(m => m.Card.TargetNumber == targetNumber).FirstOrDefault();
+                if (card is null)
+                {
+                    return false;
+                }
+
+                var ret = DeselectCardTypeB(card.Card);
+                // Deselect may fail but if selection succeed it's ok
+                ret = SelectCardTypeB(card.Card);
```

Comment:
same question, why do you need to asssing ret to DeselectCardTypeB? In here you ignore first result but in the other method you combine them

Reply:
no need, will remove it